docs/user-guide/services-firewalls.md

```diff
@@ -6,10 +6,37 @@ exposure to the internet.  When exposing a service to the external world, you ma
 one or more ports in these firewalls to serve traffic.  This document describes this process, as
 well as any provider specific details that may be necessary.
 
+### Restrict Access For LoadBlancer Service
+
+ When using a Service with `spec.type: LoadBalancer`, you can specified the IP source ranges that are allowed to access the loadbalancer
+ by using `spec.loadBalancerSourceRanges`. This field takes a list of IP CIDR ranges and kubernetes will configure firewall rules accordingly. 
```

Comment:
This field takes a list of IP CIDR ranges, which Kubernetes will use to configure firewall exceptions.